# Configuration for Hexapod initial positions

joint_trajectory_executor:
  ros__parameters:
    # Definicja nazw wszystkich stawów
    joint_names:
      - joint1_1
      - joint2_1
      - joint3_1
      - joint1_2
      - joint2_2
      - joint3_2
      - joint1_3
      - joint2_3
      - joint3_3
      - joint1_4
      - joint2_4
      - joint3_4
      - joint1_5
      - joint2_5
      - joint3_5
      - joint1_6
      - joint2_6
      - joint3_6

    step_names: [default]  # Nazwy kroków/pozycji

    # Pozycja domyślna dla wszystkich stawów
    default:
      - 0.1    # joint1_1
      - 0.5    # joint2_1
      - 1.047  # joint3_1 (60 stopni)
      - 0.0    # joint1_2
      - 0.5    # joint2_2
      - 1.047  # joint3_2 (60 stopni)
      - -0.1   # joint1_3
      - 0.5    # joint2_3
      - 1.047  # joint3_3 (60 stopni)
      - 0.1    # joint1_4
      - 0.5    # joint2_4
      - 1.047  # joint3_4 (60 stopni)
      - 0.0    # joint1_5
      - 0.5    # joint2_5
      - 1.047  # joint3_5 (60 stopni)
      - -0.1   # joint1_6
      - 0.5    # joint2_6
      - 1.047  # joint3_6 (60 stopni)

    duration: 3.0        # Czas trwania ruchu do pozycji początkowej
    epsilon: 0.01        # Tolerancja dokładności pozycji
    
    # Tematy do publikowania komend i odczytu stanu stawów
    joint_states_topic: '/joint_states'